api/v1/whisker_types.go

```diff
@@ -34,6 +40,12 @@ type Whisker struct {
 
 type WhiskerSpec struct {
 	WhiskerDeployment *WhiskerDeployment `json:"whiskerDeployment,omitempty"`
+
+	// Default: Enabled
+	// This setting enables or disables the notifications from Calico Cloud.
+	// Allowed values are Enabled or Disabled. Defaults to Enabled.
+	// +optional
+	Notifications WhiskerNotificationStatus `json:"notifications,omitempty"`
```

Comment:
```suggestion
	Notifications *WhiskerNotificationStatus `json:"notifications,omitempty"`
```

Optional fields should always be pointers

Reply:
Fixed.